### PR TITLE
using TPCH for tiflash, and using tidb-lighting to load data

### PR DESCRIPTION
**Pull request by fzhedu**
add some scripts to generate data and load schemas
uplaod the configuration files for tidb-lighting.
This is a faster way to benchmark tiflash.

### CODE REVIEW

File: tpch/dss_tiflash.sql [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+			     CONSTRAINT FOREIGN KEY LINEITEM_FK2 (L_PARTKEY,L_SUPPKEY) references partsupp(PS_PARTKEY, PS_SUPPKEY));
+
+ALTER TABLE lineitem SET TIFLASH REPLICA 1;
+ALTER TABLE orders   SET TIFLASH REPLICA 1;
+ALTER TABLE customer SET TIFLASH REPLICA 1;
+ALTER TABLE partsupp SET TIFLASH REPLICA 1;
+ALTER TABLE supplier SET TIFLASH REPLICA 1;
+ALTER TABLE part SET TIFLASH REPLICA 1;
+ALTER TABLE region SET TIFLASH REPLICA 1;
+ALTER TABLE nation SET TIFLASH REPLICA 1;
```
Comment (zz-jason) [inline tpch/dss_tiflash.sql:95]: Can we only add these SQLs? other SQLs are duplicated with another file.

File: tpch/README.md [hunk 2]
```diff
@@ -31,3 +31,12 @@ making right now, you can open another terminal and use query defined in `alltab
...
 
 If you are using macOs, please replace `#include<malloc.h>` in `varsub.c` and `bm_utils.c` to `#include<stdlib.c>`.
+
+## How to use in Tiflash, using tidb-lighting to fast load data
+
+step 1: `make dbgen`: make executable file of dbgen
+
+step 2: `sh gen_load_tiflash.sh 1 ./dss_tiflash.sql` generate data of a specific scale factor and create a database and create schema. The first parameter means the scale factor, and the second is the ddl sql. **You MUST set the right scale factor of data, and the database IP and Port in gen_load_tiflash.sh.** 
+
+step3: update `data-source-dir ` in `tidb-lightning.toml`, use it and `tikv-importer.toml` to fast load data. Update the IP and Port of these two files.
```
Comment (zz-jason) [inline tpch/README.md:41]: how about providing a link to the tidb-lightning user document?

**Comment by ilovesoup** [comment]
What is the status of this PR? Anything blocks?

File: tpch/tidb-lightning.toml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+enable = true
+schema = "tidb_lightning_checkpoint"
+
+[tikv-importer]
+# tikv-importer 的监听地址，需改成 tikv-importer 服务器的实际地址。
+addr = "172.16.5.71:8287"
+
+[mydumper]
+# Mydumper 源数据目录。
+data-source-dir = "/data1/fzh/tidb-bench/tpch/sf100"
```
Comment (ilovesoup) [inline tpch/tidb-lightning.toml:27]: Info with private path and ip should not be here.

File: tpch/tikv-importer.toml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# TiKV Importer 配置文件模版
+
+# 日志文件。
+log-file = "tikv-importer.log"
+# 日志等级：trace、debug、info、warn、error、off。
+log-level = "info"
+
+[server]
+# tikv-importer 监听的地址，tidb-lightning 需要连到这个地址进行数据写入。
+addr = "172.16.5.69:8287"
```
Comment (ilovesoup) [inline tpch/tikv-importer.toml:10]: ditto

File: tpch/tikv-importer.toml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# 给 Prometheus 客户端的推送任务名称。
+#job = "tikv-importer"
+# 给 Prometheus 客户端的推送间隔。
+#interval = "15s"
+# Prometheus Pushgateway 地址。
+#address = ""
+
+[import]
+# 存储引擎文档 (engine file) 的文件夹路径。
+import-dir = "/data1/fzh/data.import/"
```
Comment (ilovesoup) [inline tpch/tikv-importer.toml:22]: ditto

**Comment by zz-jason** [comment]
I'm going to close this PR since it hasn't been updated for a long time. Feel free to reopen it once you wish to advance this PR in the future.

**Pull request closed without merging**